[SM-703] OAS support addition (#96)
mulesoft/docs-service-mesh: modules/ROOT/pages/index.adoc

As illustrated in the diagram, at the top of the architecture is Anypoint Platform, which brings complete lifecycle API management capabilities to the microservices built with Mule runtime engine (Mule). These Mule instances might be deployed within CloudHub, or Runtime Fabric at your location. 

> Istio, which is installed on a Kubernetes Cluster, uses Envoy to manage the services using a sidecar proxy. The sidecar is responsible for any communication with the service, and resides within the same container as your service. With Anypoint Service Mesh enabled, you can continue to use Istio's native policies for traffic control and security. These policies help ensure reliable and secure service-to-service communication.

Reviewer: I would remove the final sentence, which merely restates the meaning of the sentence directly before it.
```suggestion
Istio, which is installed on a Kubernetes cluster, uses Envoy to manage services by using a sidecar proxy. The sidecar is responsible for any communication with your service, and resides within the same container as that service. With Anypoint Service Mesh enabled, you can continue to use Istio's native policies for traffic control and security. 
```
